code/modules/mob/living/say.dm

```diff
@@ -252,6 +252,8 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	var/list/the_dead = list()
 	for(var/_M in GLOB.player_list)
 		var/mob/M = _M
+		if(QDELETED(M))

```

Comment:
put a note saying this is a workaround for byond login / logout bugs, and to remove once fixed. likewise, put `see TG PR #49004` as well.

(aka likely never, but still)